src/client/components/styled-components/IconCamera.js

```diff
@@ -0,0 +1,47 @@
+                id="Shape"
+              />
+              <path
+                d="M20.710137,3.01369863 L17.289589,3.01369863 C16.9189041,3.01369863 16.5934247,2.79068493 16.4547945,2.44712329 L16.0057534,1.3230137 C15.6832877,0.518356164 14.9178082,0 14.0528767,0 L7.64575342,0 C6.78082192,0 6.01232877,0.518356164 5.69287671,1.3230137 L5.24383562,2.45013699 C5.10520548,2.79068493 4.77671233,3.01369863 4.4090411,3.01369863 L1.28986301,3.01369863 C0.578630137,3.01369863 0,3.60136986 0,4.32164384 L0,16.469863 C0,17.1931507 0.578630137,17.7778082 1.28986301,17.7778082 L20.710137,17.7778082 C21.4213699,17.7778082 22,17.190137 22,16.469863 L22,4.32164384 C22,3.60136986 21.4213699,3.01369863 20.710137,3.01369863 Z M20.7945205,16.4728767 C20.7945205,16.530137 20.7553425,16.5753425 20.710137,16.5753425 L1.28986301,16.5753425 C1.24465753,16.5753425 1.20547945,16.5271233 1.20547945,16.4728767 L1.20547945,4.32164384 C1.20547945,4.26438356 1.24465753,4.21917808 1.28986301,4.21917808 L4.4090411,4.21917808 C5.2739726,4.21917808 6.04246575,3.70082192 6.36191781,2.89616438 L6.8109589,1.7690411 C6.94958904,1.42547945 7.27506849,1.20246575 7.64575342,1.20246575 L14.049863,1.20246575 C14.4205479,1.20246575 14.7460274,1.42547945 14.8846575,1.7690411 L15.3336986,2.89315068 C15.6591781,3.70082192 16.4246575,4.21917808 17.289589,4.21917808 L20.710137,4.21917808 C20.7553425,4.21917808 20.7945205,4.26739726 20.7945205,4.32164384 L20.7945205,16.4728767 Z"
+                id="Shape"
+              />
+              <path
+                d="M17.9315068,5.12328767 C17.0183562,5.12328767 16.2739726,5.86767123 16.2739726,6.78082192 C16.2739726,7.6939726 17.0183562,8.43835616 17.9315068,8.43835616 C18.8446575,8.43835616 19.5890411,7.6939726 19.5890411,6.78082192 C19.5890411,5.86767123 18.8446575,5.12328767 17.9315068,5.12328767 Z M17.9315068,7.23287671 C17.6813699,7.23287671 17.4794521,7.0309589 17.4794521,6.78082192 C17.4794521,6.53068493 17.6813699,6.32876712 17.9315068,6.32876712 C18.1816438,6.32876712 18.3835616,6.53068493 18.3835616,6.78082192 C18.3835616,7.0309589 18.1816438,7.23287671 17.9315068,7.23287671 Z"
+                id="Shape"
+              />
+            </g>
+          </g>
+        </g>
+      </g>
+    </g>
+  </svg>
+)
+
+IconCamera.propTypes = {
+  color: PropTypes.string,
+  height: PropTypes.string,
+  width: PropTypes.string,
+}
+
+IconCamera.defaultProps = {
+  color: '#000',
```

Comment:
Is `color` a meaningful attribute, on an `svg` element? 

Regardless, it looks like we are not using it, so I'd rather we didn't add it to this file.

Reply:
Rather than removing it, I changed hardcoded color to:
`fill={props.color}`
So when we need it, it's there. We do similar things with other icons, so they should all have configurable width, height and color.